### DISCUSSION TITLE
Sharing of individual documents with users: Where can we see which users have been invited to a given document?

### DISCUSSION THREAD

**Discussion by EduNetEurope** [outline/outline#6899] [General]
My questions concerns "[Internal Sharing](https://docs.getoutline.com/s/guide/doc/sharing-LG2sGOLIpl)":
> Simply open the “Share” menu from the top of the document and then find the people to invite to the document, select, and click “Invite”. They will receive a notification and the document will appear in a “Shared with me” section in their sidebar.
> Once invited, you can also select if they have edit or view-only access to the document, or choose “No access” to remove their access.

Unfortunately, it looks like invited users need to accept the invitation before I can select if they should have edit or view-only access to a specific document.

I would like to see invited users in this dialogue even if they haven't accepted the invitation yet.
![image](https://github.com/outline/outline/assets/148647197/e12ba8b9-85c4-47bf-82e0-eec16bdb6f0f)

**Comment by tommoor** [comment]
Once the invited user is added to the document they can have the permission changed like any other user – they do not need to accept, there is no restrictions here.

**Reply by EduNetEurope** [reply]
Not working.
When I invite a user, he is not added to the menu posted above.

**Reply by tommoor** [reply]
You must search through the "Invite by name" field and add them to the document

**Reply by EduNetEurope** [reply]
Yes, and click on invite again after searching for the name. I missed that step. Thanks.

**Reply by tommoor** [reply]
Perhaps it would be clearer if the verb was "Add" in that button
